### ISSUE TITLE
NumberFormatException at Genre TypeConverter

### ISSUE THREAD

**Issue by titanwalking** [adessoTurkey/android-sample-app#39]
A NumberFormatException occurs when app tries to map an empty String into a List of Long values; since genreIds String can be an empty one.

Responsible Line: com.adesso.movee.internal.util.typeconverter.GenreConverter.fromString(GenreConverter.kt:10)

**Comment by murattuzel** [comment]
You are absolutely right about it, it should handle empty strings as well. Thanks for pointing that out. I can take the issue unless anyone else wants to.

**Comment by titanwalking** [comment]
> You are absolutely right about it, it should handle empty strings as well. Thanks for pointing that out. I can take the issue unless anyone else wants to.

Thanks Murat, appriciated, take it if you have some spare time.

**Comment by berkakbas** [comment]
I believe the issue has been solved by @titanwalking's commit on Feb 15. Shall we close it?
